fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkimport/ImportCheckPointData.java

```diff
@@ -0,0 +1,204 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.bulkimport;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.ibm.cloud.objectstorage.services.s3.model.PartETag;
+import com.ibm.fhir.model.util.FHIRUtil;
+
+// Class for tracking the partition import progress and for Batch job check points.
+// Also used as data carrier for collecting and aggregation of import metrics.
+public class ImportCheckPointData implements Serializable {
+    private static final long serialVersionUID = 2189917861035732241L;
+    // URL or COS/S3 object name.
+    private String importPartitionWorkitem;
+    private int numOfProcessedResources = 0;
```

Comment:
I believe i have by default 0 with value. So it's not necessary set 0 in this.
Document [link](https://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html)

Reply:
yes, I know this. but it doesn't harm to set it. because sometime, we get annoying warning in eclipse if you don't set any value to it before use it.